mir-module/src/main/resources/xsl/editor/mir2xeditor.xsl

```diff
@@ -129,7 +130,10 @@
         <xed:output i18n="{@label}" />
       </label>
       <div class="col-md-6 {@class}" data-type="{@type}">
-        <xsl:call-template name="mir-dateRange"/>
+        <xsl:call-template name="mir-dateRange">
+          <xsl:with-param name="showDateTimeOption" select="@showDateTimeOption"/>
```

Comment:
There is a typing issue here. Values for `showDateTimeOption` and `showDateTimeOption` from `editor-includes.xed` are attribute values and therefore of type "String", ...

... they are passed through as parameters here in the `mir:dateRange` template ...